### PR TITLE
minutes for our meeting on 20/01/2016

### PR DESCRIPTION
**Pull request by annefou**
Hi Lex & al.

Here is the draft of the minutes of our meeting (20/01/2016).

Anne.

### CODE REVIEW

File: meetings/160120_organisers.md [hunk 1]
```diff
@@ -24,3 +25,169 @@ Agenda:
...
+##Summary 2015##
+
+
+###General###
+
+The Science Library has been very helpful and helped us to run UIO Software Carpentry Workshops in 2015 (helped for the registration process, to find a room, they also handled surveys, ordered coffee/snacks). 
+
+The University of Oslo is not a member of the Software Carpentry foundation. As a consequence the University of Oslo does not have any priority on the Software Carpentry Instructor program which makes it difficult to get new instructors.
+
+USIT would be the most natural organization to become member of the Software Carpentry Foundation but they are not interested.
```
Comment (lexnederbragt) [inline meetings/160120_organisers.md]: they are not interested --> they seem not very interested

File: meetings/160120_organisers.md [hunk 2]
```diff
@@ -24,3 +25,169 @@ Agenda:
...
+
+To find another possible host for this membership, Arnstein has discussed with Knut Mørken (Computing in Science Education, Faculty of Mathematics and Natural Sciences) who showed a great interest in Software Carpentry activities; further discussions will take place to possibly include Software Carpentry lessons for bachelors. Software Carpentry may be invited for the Seminar they organized in February on reforming bachelor programs.
```
Comment (lexnederbragt) [inline meetings/160120_organisers.md]: Arnstein, Karin and Lex have discussed with Knut Mørken. organized --> organize

File: meetings/160120_organisers.md [hunk 3]
```diff
@@ -24,3 +25,169 @@ Agenda:
...
+
+Ben said that we should encourage attendees to come with colleagues as recommended by Software Carpentry. We decided to follow this recommendation and discussed on how to implement it:
+- add an item in the registration form to name possible colleagues wishing to attend the workshop
+- make sure we respect these wishes (especially when the workshop is full/nearly full and we have to pick up a few persons only).
+
+Ben mentionned recurring setup problems with attendees: several did not follow the setup instructions and did not install the necessary tools. However, we only discovered it on the second day when using git. We decided to check the availability of all the necessary tools (python, git, etc.) at the beginning of a Workshop to avoid wasting time on the second day.
+
+
+Lex also pointed an issue with the room booking system at the Science Library. It does not allow to book a room more than a month in advance which delays advertisement of Software Carpentry workshops and as a consequence the registration process. No specific action was taken to circumvent this problem.
```
Comment (lexnederbragt) [inline meetings/160120_organisers.md]: Well, what I meant was that we are allowed to use the Bikuben area at the biology buidling, but cannot ask for a reservation more than a month in advance. And in general, geeting a room for two full days is difficiult at UiO

File: meetings/160120_organisers.md [hunk 4]
```diff
@@ -24,3 +25,169 @@ Agenda:
...
+- Git: Lex
+
+Helpers: Katie, Hans (and possibly more if we can recruit a few more)
+
+
+###Actions###
+- Karin will send an email to those who are in the waiting list with a link to the registration form for the coming Software Carpentry workshop (February).
+- Karin will ask Liv to add in the registration form the possibility to name colleagues (2 or 3 maximum) wishing to attend the workshop together. 
+- Karin will modify the email sent to attendees to highlight the setup session (8:30 to 9:00 on the first day) and encourage attendees to come to the setup session if they do not manage to install the necessary software (or don't have time to do so), .
+- Anne & Hans will choose a week (no holidays, etc.) for each potential workshop  (from April) and setup a doodle for each of them.
```
Comment (lexnederbragt) [inline meetings/160120_organisers.md]: (maybe just one doodle for the whole year?)

**Comment by annefou** [comment]
Hi Lex,

Thanks for your comments/correction. I made the changes in my forked repository.
Anne.

**Pull request merged**